website/blog/2023-11-09-IPv6-addresses/item.md

```diff
@@ -0,0 +1,227 @@
+---
+title: "IPv6 addresses and why you need to make the switch now"
+description: "IPv6 addresses have been commercially available since 2010. But is there any compelling reason for sysadmins and security engineers to make the switch?"
+slug: 2023-11-09-IPv6-addresses
+authors:
+    - name: Jens Langhammer
+      title: CTO at Authentik Security Inc
+      url: https://github.com/BeryJu
+      image_url: https://github.com/BeryJu.png
+tags:
+    - authentik
+    - IP address
+    - IPv4
+    - IPv6
+    - IP address exhaustion
+    - NAT Gateway
+    - IETF
+    - Internet Engineering Task Force
+    - IANA
+    - Internet Assigned Numbers Authority
+    - IPv6 address format
+    - SSO
+    - security
+    - identity provider
+    - authentication
+hide_table_of_contents: false
+---
+
+> **_authentik is an open source Identity Provider that unifies your identity needs into a single platform, replacing Okta, Active Directory, and auth0. Authentik Security is a [public benefit company](https://github.com/OpenCoreVentures/ocv-public-benefit-company/blob/main/ocv-public-benefit-company-charter.md) building on top of the open source project._**
+
+---
+
+IPv6 addresses have been commercially available since 2010. Yet, after Google’s IPv6 rollout the following year, the adoption by System Administrators and security engineers responsible for an entire organization’s network has been slower than you might expect. Population size and the plethora of work and personal devices that accompany this large number of workers do not accurately predict which countries have deployed this protocol.
+
+In this blog post, I explain briefly what IP addresses are and how they work; share why at Authentik Security we went full IPv6 in May 2023; and then set out some reasons why you should switch now.
+
+## What are IP addresses?
+
+IP Addresses are locations (similar to street addresses) that are assigned to allow system administrators and others to identify and locate every point (often referred to as a node) on a network through which traffic and communication passes via the internet. For example, every server, printer, computer, laptop, and phone in a single workplace network has its own IP address.
+
+We use domain names for websites, to avoid having to remember IP addresses, though our readers who are sysadmin —used to referencing all sorts of nodes deep within their organization’s networks—will recall them at the drop of a hat.
```

Comment:
```suggestion
We use domain names for websites, to avoid having to remember IP addresses, though our readers who are sysadmin —used to referencing all sorts of nodes deep within their organization’s networks— will recall them at the drop of a hat.
```

Reply:
Fixed, no spaces